go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Microsoft/go-winio
 
-go 1.13
+go 1.17
```

Comment:
I guess even if this affects compat it's okay because we won't necessarily backport it, and 1.17 is the oldest supported Go release?

Reply:
Still would be nice to do things like this as a separate change, though

Reply:
Was talking to Hamza about this yesterday. The idea was to get this in as that separate change https://github.com/microsoft/go-winio/pull/230.

> I guess even if this affects compat it's okay because we won't necessarily backport it, and 1.17 is the oldest supported Go release?

Right, there was no plans to backport. We were thinking to start a v0.6.x line of tags.

Reply:
Going to resolve this as the 1.17 upgrade happened awhile ago already